airbyte-local-cli-nodejs/src/utils.ts

```diff
@@ -14,6 +14,7 @@ export const SRC_CONFIG_FILENAME = `${FILENAME_PREFIX}_src_config.json`;
 export const DST_CONFIG_FILENAME = `${FILENAME_PREFIX}_dst_config.json`;
 export const SRC_CATALOG_FILENAME = `${FILENAME_PREFIX}_src_catalog.json`;
 export const DST_CATALOG_FILENAME = `${FILENAME_PREFIX}_dst_catalog.json`;
+export const DEFAULT_STATE_FILE = 'state.json';
```

Comment:
I wonder if we should make the default state file name something that includes the src and dst image names, so it doesn't just keep overwriting the same `state.json` file every time.